### ISSUE TITLE
Changing learning rate. 

### ISSUE THREAD

**Issue by rulonboev** [SciSharp/Keras.NET#146]
Hello.

How to change learning rate for optimizer before each call model,fit ?
I need to "Annealing the learning rate"

**Comment by DoraAHsu** [comment]
You can use callback function like "ReduceLROnPlateau"

**Comment by rulonboev** [comment]
> You can use callback function like "ReduceLROnPlateau"

i'm using  c# keras api.. dont know how to callback it.

**Comment by DoraAHsu** [comment]
> > You can use callback function like "ReduceLROnPlateau"
> 
> i'm using c# keras api.. dont know how to callback it.

You can search the keyword "ReduceLROnPlateau" in keras.net sample code.
Hope it can help u.
